src/osp/tasks/top_session.h

```diff
@@ -55,6 +61,17 @@
 #define OSP_AUX_DGDI_B(x) x
 #define OSP_AUX_DGDI_A(...) OSP_AUX_DGDI_B(OSP_AUX_DGDI_C(__VA_ARGS__))
```

Comment:
If I'm reading how this macro expansion works, OSP_AUX_DCDI_A is passed OSP_DECLAR_CREATE_DATA_IDS() "session" parameter as the first arg, which stays as "session", and then "topData" becomes "count".

Maybe we need to normalize "topData" into always being named "count".

Reply:
Nevermind this, the review was hiding some important details so i misread what was going on.